src/compiler/checker.ts

```diff
@@ -4474,7 +4496,7 @@ namespace ts {
             }
         }
 
-        function getSignatureFromDeclaration(declaration: SignatureDeclaration): Signature {
+        function getSignatureFromDeclaration(declaration: SignatureDeclaration | JSDocCallbackType): Signature {

```

Comment:
why do we have to getSignatureFromDeclaration for JSDocCallbackType and not for JSDocTypedefTag?